app/controllers/providerList.js

```diff
@@ -0,0 +1,22 @@
+
+
+function info(){
+	
+var alertDialog = Titanium.UI.createAlertDialog({
```

Comment:
I was thinking if we did it this way we could add code later for the connect button to take them directly to the family they are looking at. Rather than have them close the dialog and then click something else.